README.md

```diff
@@ -1,13 +1,20 @@
 # U.S. Forms System
 
+The US Forms System is an open source library and set of guiding principles for building complex web-based forms using [React](https://reactjs.org), the [JSON Schema standard](http://json-schema.org/), and the [US Web Design System](https://designsystem.digital.gov/).
+
 ***THIS IS A WORK IN PROGRESS.***
 
-Modelled after the [US Web Design System](https://designsystem.digital.gov/), this project creates an open source code library and set of guiding principles for building complex web-based forms significantly faster than existing methods, using best practices in user experience and data collection, validation, and transmission.
+### About this project
+
+Inspired by Mozilla's [react-jsonschema-form](https://github.com/mozilla-services/react-jsonschema-form) library, the US Forms System is specifically intended for consistently styled governmental web-based forms. This library enables you to describe form fields in a JSON Schema configuration file, which then renders the backing React components necessary to build your form. You'll benefit from the US Web Design System's best practices in user experience and data collection, validation, and transmission.
```

Comment:
I think one of the main benefits of the library is how much faster you're able to build a web-based form than traditional methods, like using React directly or just using HTML and vanilla JavaScript. I wonder how we could work that in here?

Reply:
I can add something back in! 👍 Thanks for pointing it out.

Reply:
Take a look at https://github.com/usds/us-forms-system/pull/147/commits/ad005b4ed8710fb8ff91464bb7a0b7827f803a65.